### ISSUE TITLE
No progress bar while running TPOT

### ISSUE THREAD

**Issue by puskarjha** [EpistasisLab/tpot#909]
I am executing TPOT classifier and it's running fine as well. However I don't see progress bar when I execute it on my personal laptop. Same code if I execute on colab, I am getting the progress bar.

Instead of progress bar, I just receive below message

HBox(children=(IntProgress(value=0, description='Optimization Progress', max=300, style=ProgressStyle(descript…

Can someone suggest me how to get optimization progress bar ?

**Comment by flowersw** [comment]
I was running this in ubuntu/  jupyterlab - and realized this is was a tqdm issue: https://github.com/tqdm/tqdm/issues/394

I followed the instructions here to install the necessary jupyterlab exension - and it worked for me

https://ipywidgets.readthedocs.io/en/latest/user_install.html#installing-the-jupyterlab-extension

...
For example, if using conda environments, you can install nodejs with:
```
conda install -c conda-forge nodejs
```
Then you can install the labextension:
```
jupyter labextension install @jupyter-widgets/jupyterlab-manager

```

**Comment by puskarjha** [comment]
I have followed above process and have installed everything, still I am not getting progressbar. 

I am executing my code on Jupyter notebook and not JupyterLab ., if itmakes any difference.

**Comment by weixuanfu** [comment]
The issue has been fixed in new version of TPOT (v0.11.0) with updating version of tqdm. Please feel free to reopen it if you have any questions or suggestions.
